test/integration/webhook_token_auth_test.go

```diff
@@ -0,0 +1,207 @@
+package integration
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"reflect"
+	"testing"
+
+	kauthn "k8s.io/api/authentication/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	restclient "k8s.io/client-go/rest"
+	kclientcmd "k8s.io/client-go/tools/clientcmd"
+	kclientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+
+	configapi "github.com/openshift/origin/pkg/cmd/server/apis/config"
+	"github.com/openshift/origin/pkg/oc/cli/cmd"
+	userclient "github.com/openshift/origin/pkg/user/generated/internalclientset/typed/user/internalversion"
+	testutil "github.com/openshift/origin/test/util"
+	testserver "github.com/openshift/origin/test/util/server"
+)
+
+// TestWebhookTokenAuthn checks Tokens directly against an external
+// authenticator
+func TestWebhookTokenAuthn(t *testing.T) {
```

Comment:
@simo5 can you bug KC folks to get us images so the E2E tests can be written ASAP.

Reply:
not in the integration test :o)

Reply:
I mean send an email :p